src/generators/android/templates/typewriterUtils.java.hbs

```diff
@@ -0,0 +1,45 @@
+package com.segment.generated;
+
+import com.segment.analytics.Options;
+import java.util.*;
+import java.lang.String;
+import androidx.annotation.Nullable;
+import androidx.annotation.NonNull;
+
+public final class TypewriterUtils {
+  protected static Map<String, String> typewriterCtx;
+
+  static {
+    typewriterCtx = new HashMap<>();
+    typewriterCtx.put("version", "7.0.1");
```

Comment:
Can you use `{typewriterVersion}` here so that it'll automatically update the version? This'll be provided as part of the base global context: https://github.com/segmentio/typewriter/blob/f15d0369981b5c40a2dc2b04f75d01451884656a/src/generators/gen.ts#L33